src/applications/my-education-benefits/config/form.js

```diff
@@ -72,7 +76,7 @@ const formFields = {
   dateOfBirth: 'dateOfBirth',
   ssn: 'ssn',
   toursOfDuty: 'toursOfDuty',
-  toursOfDutyCorrect: 'toursOfDutyCorrect',
+  serviceHistoryIncorrect: 'serviceHistoryIncorrect',
```

Comment:
The flag name gave the impression that if it was true, the service history was correct. In fact, it was the opposite, so changed name to reflect that.